From a00be3486dcb3d14e122e2711ee11b62eea27737 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Thu, 11 Oct 2018 12:15:51 +0300
Subject: [PATCH 321/706] arm64: dts: ls1028a: Drop phy nodes fron PXP dts as
 link bringup is too slow for PXP

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit cb47b80949387c5b9f2305a7b7cb2fa81bc7b0f2)
(cherry picked from commit b203bd588933cf98d16dfdfcc458b7f5766230af)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 .../boot/dts/freescale/fsl-ls1028a-pxp.dts    | 32 -------------------
 1 file changed, 32 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-pxp.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-pxp.dts
index 5e01cfa48a0e..2a710544d91a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-pxp.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-pxp.dts
@@ -61,38 +61,6 @@
 	status = "okay";
 };
 
-&enetc_port0 {
-	phy-connection-type = "internal";
-	fixed-link {
-		speed = <1000>;
-		full-duplex;
-	};
-};
-
-&enetc_port1 {
-	phy-connection-type = "internal";
-	fixed-link {
-		speed = <1000>;
-		full-duplex;
-	};
-};
-
-&enetc_port2 {
-	phy-connection-type = "internal";
-	fixed-link {
-		speed = <1000>;
-		full-duplex;
-	};
-};
-
-&enetc_port3 {
-	phy-connection-type = "internal";
-	fixed-link {
-		speed = <1000>;
-		full-duplex;
-	};
-};
-
 /* l2switch ports */
 &switch_port0 {
 	phy-connection-type = "internal";
-- 
2.17.1

